composer.json

```diff
@@ -17,12 +17,12 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8",
+        "php": ">=7.1 <8.2",
```

Comment:
OK, I will remove.  I just like to be more strict.  What happens is you can silently break someone's system by allowing them to upgrade their PHP version, but the package has incompatibilities with the newer PHP version. This is what happened to me. I did not notice DebugBar was throwing warnings since I don't normally run it. But then I did, and it was not working, leading to this PR.

I think it is better for someone to know the package has not been vetted on a newer version of PHP when Composer will not upgrade them.  Since it is a preexisting problem, this would just fix it going forward. So what would happen after someone upgrades to 8.2, Composer will downgrade DebugBar to the version before this.  So maybe at some point once we are sure it is 8.1 compatible, we can make this change.  Then when they upgrade to 8.2, they will get a message from Composer.  At that point, the package can be updated if there are 8.2 incompatibilities.